### PR TITLE
RachelSassHW01

### PR DESCRIPTION
**Pull request by rsasschicago**
Overall, this assignment went relatively smoothly.

First, I forked hw01 from the course GitHub and then copied it to my RStudio account, saving it in a folder titled "homework."

I then added text to the README file, staged and committed the changes, and pushed them back to my GitHub account (using my account URL). 

After adding some more text and an image, and committing and pushing two more times along the way, I returned to my GitHub repository and created this pull request.

I ran into issues when I tried to commit/push from my local RStudio because the app is connected to an old GitHub account that I no longer use. 

On the browser version of RStudio, I was unable to figure out how to use an image downloaded on my computer in my R Markdown, so instead I used a link to a web version of the image. 

The last issue I ran into was that in the preview of my Markdown on GitHub, my addition of bold text was not displaying.

### CODE REVIEW

**Review by emilyrobb — COMMENTED** [review]
Hi Rachel, this looks awesome! Here are my comments:

1. Great use of different markdown heading sizes!

1. Awesome image use -- so pretty.

1. Another comment about the image -- maybe consider sizing it differently depending on what you want to emphasize! But overall great job.

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
 # hw01
+
+# My name is **Rachel Sass**. Here are a few things about me!
```
Comment (deblnia) [inline README.md:3]: This bolding won't render because of the heading (the heading takes precedence).

File: README.md [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+## A bit about me
+
+#### I am from Irvine, California, and I went to college at UCLA. My majors were History and Political Science. Last year, I lived in Washington, DC, where I worked at a non-profit that provides pro bono legal services to residents with debt crises. I'm hoping to do policy research in the future, so I'm excited to gain social science computing skills in this class.
```
Comment (deblnia) [inline README.md:7]: I'd just use plain text here.

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       X       |            |
| **Coding strategy**         |           |    X          |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |    X       |              |            |
| **Reproducibility**         |     X      |              |            |

Remarks:

* Overall, solid work! You've used four elements of of markdown syntax successfully, and you seem comfortable with the git workflow, both of which are great. 
* When you're writing using any kind of markup, it's good practice to render your final product to make sure it looks the way you intended. (You might have caught the lack of bold by doing that.)

**Comment by anariaah** [comment]
1. Your layout is clean and straightforward! You use two sizes of headers to make it clear!
2. Your image is pretty but the size might be a bit large.

**Pull request closed without merging**